src/savegame_json.cpp

```diff
@@ -3808,6 +3823,25 @@ void submap::load( JsonIn &jsin, const std::string &member_name, bool rubpow_upd
             }
             partial_constructions[pt] = pc;
         }
+    } else if( member_name == "crafting_bills" ) {
+        jsin.start_array();
+        while( !jsin.end_array() ) {
+            crafting_bill cb;
+            int i = jsin.get_int();
+            int j = jsin.get_int();
+            int k = jsin.get_int();
+            tripoint pt = tripoint( i, j, k );
+            jsin.start_array();
+            std::map<recipe_id, int> loaded_bill;
+            while( !jsin.end_array() ) {
+                recipe_id tmp;
+                jsin.read( tmp );
+                int count = jsin.get_int();
+                loaded_bill[tmp] = count;
```

Comment:
```suggestion
                cb.bills[tmp] = count;
```
There is no need to load this into a temporary map.

In fact, the whole loop body can be simpified:
```C++
cb.bills.emplace( recipe_id( jsin.get_string() ), jsin.get_int() );
```